### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by jrycw**

### CODE REVIEW

File: listings/ch19-advanced-features/listing-19-21/output.txt [hunk 1]
```diff
@@ -2,4 +2,4 @@ $ cargo run
    Compiling traits-example v0.1.0 (file:///projects/traits-example)
     Finished dev [unoptimized + debuginfo] target(s) in 0.48s
      Running `target/debug/traits-example`
-幼犬被稱作小狗崽
```
Comment (weihanglo) [inline listings/ch19-advanced-features/listing-19-21/output.txt:5]: listing 19-21 is still "小狗崽". Could you find and replace all 小狗崽 to 小狗狗 in this repo?

https://github.com/rust-tw/book-tw/blob/394f88d0eb270c5be1c68931e0186fe09f0ec854/listings/ch19-advanced-features/listing-19-21/src/main.rs#L15

File: src/ch02-00-guessing-game-tutorial.md [hunk 2]
```diff
@@ -423,7 +423,7 @@ anchor or snip comments
 let guess: u32 = guess.trim().parse().expect("請輸入一個數字！");
 ```
 
-我們建立了一個變數叫做 `guess`。小等一下，程式不是已經有個變數叫做 `guess`了嗎？的確是的，但 Rust 允許我們遮蔽之前的 `guess` 數值成新的數值。**遮蔽**（Shadowing）讓我們可以重複使用 `guess` 變數名稱，而不必強迫我們得建立兩個不同的變數，舉例來說像是 `guess_str` 和 `guess`。我們會在[第三章][shadowing]<!-- ignore -->更詳細地解釋此概念，現在這邊只需要知道這常拿來將一個數值的型別轉換成另一個型別。
+我們建立了一個變數叫做 `guess`。稍等一下，程式不是已經有個變數叫做 `guess`了嗎？的確是的，但 Rust 允許我們遮蔽之前的 `guess` 數值成新的數值。**遮蔽**（Shadowing）讓我們可以重複使用 `guess` 變數名稱，而不必強迫我們得建立兩個不同的變數，舉例來說像是 `guess_str` 和 `guess`。我們會在[第三章][shadowing]<!-- ignore -->更詳細地解釋此概念，現在這邊只需要知道這常拿來將一個數值的型別轉換成另一個型別。
```
Comment (weihanglo) [inline src/ch02-00-guessing-game-tutorial.md]: I believe this is not a typo. It's Taiwanese. I was not the translator for this paragraph though.

https://sutian.moe.edu.tw/zh-hant/su/455/

**Review by weihanglo — COMMENTED** [review]
Thanks for the contribution.

Just replied with some nitpicks.

File: listings/ch19-advanced-features/listing-19-21/output.txt (hunk 1 above)
Comment (jrycw) [reply listings/ch19-advanced-features/listing-19-21/output.txt:5]: Thanks for the reminder. I found another 2 "小狗崽" and replaced them with "小狗狗".

File: src/ch02-00-guessing-game-tutorial.md (hunk 2 above)
Comment (jrycw) [reply src/ch02-00-guessing-game-tutorial.md]: Oh, thank you. I didn't realize this is Taiwanese. I've reverted the update.

**Review by weihanglo — APPROVED** [review]
Thank you for the patch!

**Pull request merged**